tests/lib/openstack_query/test_query_methods.py

```diff
@@ -151,7 +151,7 @@ def test_to_list_as_objects_true(self):
         """
         mock_query_results_list = ["object-1", "object-2"]
 
-        # pylint: disable=too-few-public-methods
+        # pylint: disable=protected-access
```

Comment:
Why has this changed to protected-access, even though nothing else has changed in the file?